src/main/java/taxi/web/filters/AuthenticationFilter.java

```diff
@@ -0,0 +1,43 @@
+package taxi.web.filters;
+
+import java.io.IOException;
+import java.util.HashSet;
+import java.util.Set;
+import javax.servlet.Filter;
+import javax.servlet.FilterChain;
+import javax.servlet.FilterConfig;
+import javax.servlet.ServletException;
+import javax.servlet.ServletRequest;
+import javax.servlet.ServletResponse;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+public class AuthenticationFilter implements Filter {
+    private static final String DRIVER_ID = "driver_id";
+    private static final Set<String> ALLOWED_URLS = new HashSet<>();
```

Comment:
```suggestion
    private static final Set<String> allowedUrls = new HashSet<>();
```
it's not "deeply immutable" so we don't use constant naming